.travis.yml

```diff
@@ -93,6 +97,9 @@ matrix:
       - dist: trusty
         env:
           - JOB="3.6_NUMPY_DEV" TEST_ARGS="--skip-slow --skip-network" PANDAS_TESTING_MODE="deprecate"
+      - dist: trusty
```

Comment:
same here

Reply:
@jorisvandenbossche prefered this before the doc build

https://github.com/pandas-dev/pandas/pull/19236#discussion_r162569822